.github/workflows/_binary-build-linux.yml

```diff
@@ -16,6 +16,13 @@ on:
           default: linux.12xlarge
           type: string
           description: Hardware to run this "build"job on, linux.12xlarge or linux.arm64.2xlarge.
+      use_split_build:
```

Comment:
What do you think about give the split build a more specific name in code?

E.g. "split_libtorch_build" or "use_libtorch_wheel"